packages/vscode-extension/src/utilities/sdkmanager.ts

```diff
@@ -79,7 +81,9 @@ export async function getAndroidSystemImages(): Promise<AndroidSystemImageInfo[]
 
 // example input: 'android-34/default/arm64-v8a/data'
 function mapToSystemImageInfo(systemImagePath: string) {
-  const [imageName, systemImageType, arch] = systemImagePath.split("/");
+  const [imageName, systemImageType, arch] = (process.platform === "win32") 
+    ? systemImagePath.replace(SYSTEM_IMAGES_PATH + path.sep, "").split(path.sep)
```

Comment:
How's the input look on windows?

Reply:
sample input looks like 'C:\\Users\\USER\\AppData\\Local\\Android\\Sdk\\system-images\\android-35\\google_apis_playstore\\x86_64\\data' so we have to truncate the beginning.

Reply:
It shouldn't look like that. Can you please change the truncation code in the `recursiveSystemImagePathsSearch`? It hardcodes unix separator.

Reply:
You're right. That's much simpler.